python/packages/ai/teams/ai/clients/llm_client.py

```diff
@@ -284,7 +291,20 @@ def chunk_received(
 
             if streamer is not None:
                 await streamer.end_stream()
+                
+                # Tool call handling
+                # Keep the streamer around during tool calls, letting them return as normal messages minus the content.
+                # When the tool call completes, reattach to the streamer for continued streaming to the client.
+                if res.message and isinstance(res.message.action_calls, list):
```

Comment:
instead of checking that its a list, could we check that its not None, and length is greater than 0?

Reply:
sure I'll try this.